pitch.md

```diff
@@ -57,13 +82,17 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+It seems that trying to invent ways for the user to specify how they want to build the map in an intuitive way should involve more language design decisions. There will be semantics decisions, of course, but trying to add in as much implementation as possible should draw out the language aspects of this project.
```

Comment:
It seems like there could be a large amount of implementation work if you wanted to do something like making sure the water area when tiled looks like. If someone wanted to tile an area with water, they would have to design their tiles to match up with the tiles next to them, which sounds really frustrating. It would be cool if your project automatically tweaked the tiling and/or images so that they lined up well.